mobile/src/main/java/org/openhab/habdroid/ui/oneitemactivities/AbstractOneItemActivity.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.habdroid.ui.oneitemactivities
+
+import org.openhab.habdroid.ui.AbstractBaseActivity
+
+/**
+ * Base class for all activities that display a single Item or Widget.
+ */
+abstract class AbstractOneItemActivity : AbstractBaseActivity()
```

Comment:
What's the point of this base class? 

Also I don't really like the package name, since it's both misleading (chart and image activities deal with widgets and not items), and somewhat hard to read.

Reply:
I'd like to have some kind of grouping of activities that handle a single item or widget, e.g by package, base class or naming convention.

What about `Single(Image|Chart|Color)(Item|Widget)Activity` as name and no base class?

Reply:
That's OK for me. I'm not sure the `Single` prefix gains us much (e.g. `ChartWidgetActivity` sounds clear enough for me, `SingleChartWidgetActivity` IMO just makes it longer), but I don't have strong feelings either way.